### PR TITLE
Swarm docs: Fix typo problem, 'swarm1' to 'agent1'

### PR DESCRIPTION
**Pull request by lanphan**
Just a minor PR, please help to review.

### CODE REVIEW

**Comment by abronan** [comment]
Thanks @lanphan, can you please `sign-off` your commit though before we merge? You can find the instructions [here](https://docs.docker.com/v1.7/docker/project/review-pr/), section *Updating an existing Pull Request*, use `-s` when you commit :wink:. Thanks!

File: docs/install-w-machine.md [hunk 1]
```diff
@@ -177,7 +177,7 @@ Here, you connect to the cluster and review information about the Swarm manager
         .
         .
 
-    In this case, the Swarm ran 'hello-world' on the 'swarm1'.
+    In this case, the Swarm ran 'hello-world' on the 'agent1'.
```
Comment (nishanttotla) [inline docs/install-w-machine.md]: While you're editing this line, could you also update the hello-world and agent1 to markdown syntax using backticks?
Like `hello-world` and `agent1`. Thanks!

**Comment by lanphan** [comment]
@abronan done, please help to recheck

**Comment by nathanleclaire** [comment]
@lanphan It needs a rebase please.

**Comment by lanphan** [comment]
Hi @nathanleclaire ,
It seems that other have fixed it, I closed this one.

**Pull request closed without merging**